sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -25,38 +25,45 @@ static dif_gpio_t gpio;
 static dif_rv_timer_t timer;
 static dif_uart_t uart;
 
-// UART input maximum buffer sizes
-static const uint32_t kMaxInputLengthText = 128;
-static const uint32_t kMaxInputLengthBin = 64;
-
-// AES Configuration parameters. These values must match the test driving
-// side.
-static const uint32_t kAesBlockSize = 16;
-static const uint32_t kAesKeySize = 16;
+enum {
+  kAesKeySize = 16,
```

Comment:
Please rename this to `kAesKeyLength`. Key length is the common term used at least in the AES world.